webrender/src/render_backend.rs

```diff
@@ -214,9 +214,13 @@ impl RenderBackend {
                             }
                         }
                         ApiMsg::SetRootPipeline(pipeline_id) => {
-                            let frame = profile_counters.total_time.profile(|| {
-                                self.scene.set_root_pipeline_id(pipeline_id);
+                            self.scene.set_root_pipeline_id(pipeline_id);
+
+                            if self.scene.display_lists.get(&pipeline_id).is_none() {
```

Comment:
uber-nit: could use `!self.scene.display_lists.contains_key(&pipeline_id)`
it's totally irrelevant here, I know :)